src/main/java/software/amazon/awssdk/crt/http/HttpRequest.java

```diff
@@ -25,41 +26,42 @@
 
     /**
      *
-     * @param method http verb to use
+     * @param method      http verb to use
      * @param encodedPath path of the http request
      */
     public HttpRequest(String method, String encodedPath) {
-        this(method, encodedPath, new HttpHeader[]{}, null);
+        this(method, encodedPath, new HttpHeader[] {}, null);
     }
 
     /**
      *
-     * @param method http verb to use
+     * @param method      http verb to use
      * @param encodedPath path of the http request
-     * @param headers set of http request headers to include
-     * @param bodyStream (optional) interface to an object that will stream out the request body
+     * @param headers     set of http request headers to include
+     * @param bodyStream  (optional) interface to an object that will stream out the
+     *                    request body
      */
     public HttpRequest(String method, String encodedPath, HttpHeader[] headers, HttpRequestBodyStream bodyStream) {
-        if (headers == null) { throw new IllegalArgumentException("Headers can be empty, but can't be null"); }
+        if (headers == null) {
+            throw new IllegalArgumentException("Headers can be empty, but can't be null");
+        }
         this.method = method;
         this.encodedPath = encodedPath;
         this.headers = Arrays.asList(headers);
         this.bodyStream = bodyStream;
     }
 
     /**
-     * Package private. Used by JNI to convert native http representation to a Java object, because accessing this
-     * struct from JNI is too slow.
+     * Package private. Used by JNI to convert native http representation to a Java
+     * object, because accessing this struct from JNI is too slow.
      *
-     *  Requests are marshalled as follows:
+     * Requests are marshalled as follows:
      *
-     *  each string field is:
-     *  [4-bytes BE] [variable length bytes specified by the previous field]
-     *  Each request is then:
-     *  [method][path][header name-value pairs]
+     * each string field is: [4-bytes BE] [variable length bytes specified by the
```

Comment:
All this reformatting makes it quite a bit harder to actually see the changes and in some cases reduces the readability (like here).  We don't really do any active formatting on the Java/JS/Python side as far as I know.

If we want to do that, it should be a separate PR.